pipelines/rj_crm__disparo_template/utils/dispatch.py

```
@@ -878,6 +974,101 @@ def get_failed_cpfs(billing_project_id: str, id_hsm: int,) -> set:
+    Usada pelo monitoramento pós-SFTP para decidir se deve alertar no Discord por falta
+    de confirmação de entrega.
+    """
+    query = f"""
+        SELECT COUNT(*) AS total_sucesso
+        FROM `rj-crm-registry.brutos_salesforce.status_disparo`
+        WHERE LOWER(nome_hsm) = LOWER('{campaign_name}')
+          AND envio_datahora >= '{dispatch_date}'
+          AND indicador_falha = FALSE
+          AND indicador_quarentena = FALSE
+          AND entrega_datahora IS NOT NULL
+    """
```

Ruff (0.15.21)

[error] Possible SQL injection vector through string-based query construction

(S608)